src/spikeinterface/postprocessing/tests/test_correlograms.py

```diff
@@ -14,54 +12,47 @@
 from spikeinterface.postprocessing.tests.common_extension_tests import AnalyzerExtensionCommonTestSuite
 from spikeinterface.postprocessing import ComputeCorrelograms
 from spikeinterface.postprocessing.correlograms import compute_correlograms_on_sorting, _make_bins
+import pytest
 
 
-class ComputeCorrelogramsTest(AnalyzerExtensionCommonTestSuite, unittest.TestCase):
-    extension_class = ComputeCorrelograms
-    extension_function_params_list = [
-        dict(method="numpy"),
-        dict(method="auto"),
-    ]
-    if HAVE_NUMBA:
-        extension_function_params_list.append(dict(method="numba"))
+class TestComputeCorrelograms(AnalyzerExtensionCommonTestSuite):
+
+    @pytest.mark.parametrize(
+        "params",
+        [
+            dict(method="numpy"),
+            dict(method="auto"),
+            pytest.param(dict(method="numba"), marks=pytest.mark.skipif(not HAVE_NUMBA, reason="Numba not available")),
+        ],
+    )
+    def test_extension(self, params):
+        self.run_extension_tests(ComputeCorrelograms, params)
 
 
 def test_make_bins():
+    """
+    Check the `_make_bins()` function that generates time bins (lags) for
+    the correllogram creates the expected number of bins.
+    """
     sorting = generate_sorting(num_units=5, sampling_frequency=30000.0, durations=[10.325, 3.5], seed=0)
 
     window_ms = 43.57
     bin_ms = 1.6421
     bins, window_size, bin_size = _make_bins(sorting, window_ms, bin_ms)
     assert bins.size == np.floor(window_ms / bin_ms) + 1
-    # print(bins, window_size, bin_size)
 
     window_ms = 60.0
     bin_ms = 2.0
     bins, window_size, bin_size = _make_bins(sorting, window_ms, bin_ms)
     assert bins.size == np.floor(window_ms / bin_ms) + 1
-    # print(bins, window_size, bin_size)
 
 
 def _test_correlograms(sorting, window_ms, bin_ms, methods):
     for method in methods:
         correlograms, bins = compute_correlograms_on_sorting(sorting, window_ms=window_ms, bin_ms=bin_ms, method=method)
         if method == "numpy":
-            ref_correlograms = correlograms
             ref_bins = bins
         else:
-            # ~ import matplotlib.pyplot as plt
-            # ~ for i in range(ref_correlograms.shape[1]):
-            # ~ for j in range(ref_correlograms.shape[1]):
-            # ~ fig, ax = plt.subplots()
-            # ~ ax.plot(bins[:-1], ref_correlograms[i, j, :], color='green', label='numpy')
-            # ~ ax.plot(bins[:-1], correlograms[i, j, :], color='red', label=method)
-            # ~ ax.legend()
-            # ~ ax.set_title(f'{i} {j}')
-            # ~ plt.show()
```

Comment:
Having mpl commented for debugging is sometimes convenient.
But it is Ok I will but then again next time I debug theses :)